pkg/util/podvolume/pod_volume.go

```diff
@@ -29,6 +29,32 @@ import (
 	"github.com/vmware-tanzu/velero/pkg/util"
 )
 
+func ShouldPerformFSBackupLegacy(
```

Comment:
Lets remove this function as it is not being used.

Reply:
Thanks. Removed.